src/main/webapp/dev/js/main/feedbackSubmissionsEdit.es6

```diff
@@ -963,10 +963,12 @@ $(document).ready(() => {
     if (typeof richTextEditorBuilder !== 'undefined') {
         $.each(textFields, (i, textField) => {
             const id = $(textField).attr('id');
+            const isSessionClosed = $(textField).data('is-session-closed') || false;
 
             /* eslint-disable camelcase */ // The property names are determined by external library (tinymce)
             richTextEditorBuilder.initEditor(`#${id}`, {
                 inline: true,
+                readonly: isSessionClosed,
```

Comment:
@whipermr5 I understand your reservations about passing `IS_SESSION_CLOSED` and I agree that your last statement would be the most readable. @amarlearning Go ahead with @whipermr5's suggestions. I have nothing more to add :)